compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -183,6 +183,40 @@ TR::Instruction *loadConstant64(TR::CodeGenerator *cg, TR::Node *node, int64_t v
    return cursor;
    }
 
+int32_t getFP8bitImm(double value)
```

Comment:
I don't think this function needs to live in OMRTreeEvaluator.cpp because it is called only from fconst/dconstEvaluator().
Please move the function to FPTreeEvaluator.cpp and make it a static function.